src/containers/Review/ReviewPageWrapper.tsx

```diff
@@ -17,6 +20,24 @@ const ReviewPageWrapper: React.FC = () => {
     serialNumber,
   })
 
+  const [updateReviewResponse] = useUpdateReviewResponseMutation({
+    onCompleted: ({ updateReviewResponse }) =>
+      console.log('Success to update reviewResponse: ', updateReviewResponse?.clientMutationId),
+    onError: (error) => console.log('Problem to update reviewResponse', error.message),
```

Comment:
```suggestion
    onError: (error) => console.log('Problem updating reviewResponse', error.message),
```
Just my picky English 😜